client/app/src/accounts/AccountController.js

```diff
@@ -129,6 +129,20 @@
       else return null;
     };
 
+    self.clearData = function() {
+      var confirm = $mdDialog.confirm()
+            .title('Are you sure?')
+            .textContent('All your data, including created accounts, networks and contacts will be lost and reset to default.')
```

Comment:
also it could be good to have it with translation:
see https://github.com/ArkEcosystem/ark-desktop/blob/master/client/app/src/accounts/AccountController.js#L117